org.apache.airavata.django.tests/src/AbinitComet.java

```diff
@@ -0,0 +1,57 @@
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Add Apache license header as in https://github.com/apache/airavata/blob/master/modules/airavata-helix/helix-spectator/src/main/java/org/apache/airavata/helix/impl/workflow/PostWorkflowManager.java#L1